weaviate/collections/grpc/query.py

```diff
@@ -1072,6 +1072,7 @@ def add_vector(val: Sequence[float], target_name: str) -> None:
                             ],
                         )
                     )
+                    target_vectors_tmp.append(key)
```

Comment:
if this works, then couldn't we just return `list(vector.keys())` on line 1085 instead of `target_vectors_tmp` (and get rid of `target_vectors_tmp`)?